js/app/packages/app/index.css

```
@@ -285,6 +310,28 @@
+@keyframes breathe {
+  0%, 100% { transform: scale(1); }
+  50%      { transform: scale(1.05); }
+}
+
+@keyframes pulse-ring {
+  0%       { transform: scale(1); opacity: 0.5; }
+  50%      { transform: scale(1.15); opacity: 0; }
+  100%     { transform: scale(1); opacity: 0; }
+}
+
+@keyframes fadeSlideIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)